backend/src/services/new-cars/new-cars.service.ts

```diff
@@ -0,0 +1,77 @@
+import { prisma } from '@data/prisma-client';
+
+import type { ModelResponseDto } from '@autoline/shared/common/types/types';
+
+const getNewCar = async (
+  take: number,
+  skip: number,
+): Promise<ModelResponseDto> => {
+  const newcar = await prisma.model.findMany({
+    skip,
+    take,
```

Comment:
We need to have a 1-1 talk about this function. It's really wrong on a few levels at the same time